TwitchPlaysAssembly/Src/ComponentSolvers/ComponentSolver.cs

```diff
@@ -666,6 +671,13 @@ protected void HandleModuleException(Exception e)
 		SolveModule($"Looks like {Module.BombComponent.GetModuleDisplayName()} ran into a problem while running a command, automatically solving module.");
 	}
 
+	protected void HandleNRSolve(string reason)
+	{
+		reason = reason.Remove(0, 7).Trim(' ');
```

Comment:
Calling `.Trim()` with no arguments removes all whitespace characters. I don't think most people would use a tab or the Unicode whitespace characters but covering those is good to do.